.travis.yml

```diff
@@ -16,7 +16,17 @@ install: "make"
 script:
   - |
     if [[ "$TRAVIS_PYTHON_VERSION" != "2.6" ]] ; then make test-readme; fi
-  - make coverage
-after_success:
-  - pipenv run codecov
+  - make ci
 cache: pip
+jobs:
```

Comment:
Oh, this is actually the same on master though, so it's probably worth fixing there.